src/tools/brush/utils/Operations/fillInside.js

```diff
@@ -0,0 +1,46 @@
+import { getBoundingBoxAroundPolygon } from '../Boundaries';
+import pointInPolygon_ from '../../../../util/freehand/pointInFreehand.js';
+import { pointInPolygon } from '../PointInside';
+
+import { getLogger } from '../../../../util/logger.js';
+
+const logger = getLogger('tools:ScissorsTool');
+
+export default function fillInside(
+  points,
+  segmentationData,
+  image,
+  labelValue = 1
+) {
+  // Loop through all pixels in the segmentation data mask
+
+  // Obtain the bounding box of the entire drawing so that
+  // we can subset our search. Outside of the bounding box,
+  // everything is outside of the polygon.
+  const { width } = image;
+  const vertices = points.map(a => [a.x, a.y]);
+  const [topLeft, bottomRight] = getBoundingBoxAroundPolygon(vertices, image);
+
+  const [xMin, yMin] = topLeft;
+  let [xMax, yMax] = bottomRight;
+
+  let painted = 0;
```

Comment:
I know we briefly touched on this and before, and we can keep it if you are about to use it for anything, but `painted` it only ever incremented and never evaluated, so in the current state of the tool I don't think it should exist. Same case in `fillOutside.js` and thus also `fillOutsideBoundngBox.js`

Reply:
Yeah this was my sanity check to see how many pixels were being changed. It should be removed.

Reply:
All painted loggers removed successfully.